assets/terraform/gce/bootstrap/suse.sh

```diff
@@ -96,24 +111,31 @@ fi
 # grow-root-fs expands '/' to use all available space on the device.
 #
 # The GCP sles-12-sp5-v20200610 image doesn't recognize extra space in drives
-# by default, and limits the '/' partition to 10G without this kick.
+# by default, and limits the '/' partition to 10G without this kick. This doesn't
+# affect suse-cloud/sles-15-sp1-v20200415.
 #
 # See https://cloud.google.com/compute/docs/disks/add-persistent-disk#resize_partitions
 function grow-root-fs {
-  echo "Filesystem utilization before grow-root-fs:"
-  df -h
 
   local root_fs_dev=$(findmnt --noheadings -o SOURCE /) # e.g. /dev/sda3
   local root_fs_dev_name=$(lsblk --noheadings -o kname $root_fs_dev) # e.g. sda3
   local partition_number=$(cat /sys/class/block/$root_fs_dev_name/partition) # e.g. 3
   local parent_dev_name=$(lsblk --noheadings -o pkname $root_fs_dev) # e.g. sda
   local parent_dev=/dev/$parent_dev_name # e.g. /dev/sda
 
-  growpart $parent_dev $partition_number
-  xfs_growfs /
-
-  echo "Filesystem utilization after grow-root-fs:"
-  df -h
+  local root_fs_size_bytes=$(sudo blockdev --getsize64 $root_fs_dev)
+  local root_fs_size_gb=$((root_fs_size_bytes / (1024 * 1024 * 1024)))
+  local expected=60
```

Comment:
This should probably be tied to https://github.com/gravitational/robotest/blob/8e83284206d4acd004b50cdf0812034a14c61585/assets/terraform/gce/node.tf#L108 instead of having both as magic numbers

Reply:
Addressed in 7fb57ae.